cli/flox/src/utils/dialog.rs

```diff
@@ -58,6 +58,23 @@ impl<'a> Dialog<'a, Confirm> {
         .await
         .expect("Failed to join blocking dialog")
     }
+
+    pub fn prompt_sync(self) -> inquire::error::InquireResult<bool> {
```

Comment:
question: this is unused, did you mean to include it?

Reply:
I dirst implemented this with a confirm and took the opportunity to start "syncify" these prompt impls..
I can remove this here, bring it back when we stumble over it next time

Reply:
removed, we'll fish it back out next time